pkg/search/query.go

```diff
@@ -383,7 +384,7 @@ func (c *Constraint) matchesAtMostOneBlob() blob.Ref {
 }
 
 func (c *Constraint) onlyMatchesPermanode() bool {
-	if c.Permanode != nil || c.CamliType == "permanode" {
+	if c.Permanode != nil || c.CamliType == string(schema.PermanodeCamliType) {
```

Comment:
likewise, why not just the CamliType type?

Reply:
Done